.github/workflows/python-standard.yml

```diff
@@ -0,0 +1,31 @@
+name: Github Actions Python Standard
+
+on: [push]
```

Comment:
should also do on pull requests

Reply:
Noted 👌, also to clarify, it would be on any branch then right?

Reply:
yeah, typically a good idea. we do this in most of the other open source repositories

Reply:
Gotcha, will update it to PR's as well